pkg/kates/apply.go

```diff
@@ -0,0 +1,23 @@
+package kates
+
+import (
+	"context"
+	"os"
+
+	"k8s.io/kubectl/pkg/cmd/apply"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+// IncoherentApply is like `kubectl apply`, and DOES NOT count as a "write" for the purposes of the
+// Client's read/write coherence.
+func (c *Client) IncoherentApply(ctx context.Context, stdio IOStreams, args ...string) error {
```

Comment:
You implement this but I didn't see it being called anywhere? is it implementing an interface function that is called as part of the deprecate k8s machinary?

Reply:
It's called from `kubeapply.go` in 4fae46776d028bb2ce204f729196985e805166cd.

Reply:
I see it now. Thanks. I knew I was probably just missing it 😄 .